drivers/display/display_stm32_ltdc.c

```diff
@@ -260,6 +262,40 @@ static int stm32_ltdc_read(const struct device *dev, const uint16_t x,
 	return 0;
 }
 
+static int stm32_ltdc_display_blanking_off(const struct device *dev)
+{
+	const struct display_stm32_ltdc_config *config = dev->config;
+	const struct device *display_dev = config->display_controller;
```

Comment:
`config->display_controller` is ambiguous in the context of the device controller driver, needs to be improved, maybe `ext_dc` or `panel_controller`.